app/assets/javascripts/govuk-component/feedback.js

```diff
@@ -0,0 +1,147 @@
+(function (Modules) {
+  "use strict";
+  window.GOVUK = window.GOVUK || {};
+
+  Modules.Feedback = function () {
+
+    this.start = function ($element) {
+      this.$prompt = $element.find('.js-prompt');
+      this.$fields = $element.find('.pub-c-feedback__form-field');
+      this.$forms = $element.find('.js-feedback-form');
+      this.$toggleForms = $element.find('.js-toggle-form');
+      this.$closeForms = $element.find('.js-close-form');
+      this.$activeForm = false;
+      this.$pageIsUsefulButton = $element.find('.js-page-is-useful');
+      this.$pageIsNotUsefulButton = $element.find('.js-page-is-not-useful');
+      this.$somethingIsWrongButton = $element.find('.js-something-is-wrong');
+      this.$promptQuestions = $element.find('.js-prompt-questions');
+      this.$promptSuccessMessage = $element.find('.js-prompt-success');
+
+      var that = this;
+
+      setInitialAriaAttributes();
+
+      this.$toggleForms.on('click', function(e) {
```

Comment:
Can we reuse any of https://github.com/alphagov/static/blob/master/app/assets/javascripts/modules/toggle.js to do the toggling?

Reply:
It's slightly more complicated, because not only are we showing a form, we're also hiding the thing that was clicked on, and setting focus, and firing GA events. Is all that possible with toggle.js?